docs/content/developers/building-contributing.md

```diff
@@ -135,6 +135,37 @@ To manually push using GitHub Actions,
 * Click “Run workflow” in the blue band near the top.
 * Choose the branch, usually `master`. Include a tag for the pushed image and GitHub will do all the work.
 
+## Instrumentation
+
+The instrumentation implementation is generated using the [Ampli Codegen](https://www.docs.developers.amplitude.com/data/sdks/go/ampli/).
+
+To synchronize the implementation with the latest changes at Amplitude, the CLI
+tool has to be installed locally:
+
+```bash
+npm install -g @amplitude/ampli
+```
+
+Make changes to the event definition using the GUI at <https://data.amplitude.com/ddev/DDEV>:
+
+* create a new branch
+* create or change events and properties
+* save changes to the new branch
+* update the implementation with `ampli checkout <branch name>`
+* make changes to the code
+
+Once finished, save the changes to publish a new version of the definitions.
+
+Afterwards the changes can be imported running the following command in the
+project root:
+
+```bash
+ampli pull
+```
+
+Make sure the API keys are not included to the sources; they are linked during
+compilation using a GitHub secret.
```

Comment:
What is the current status of the PR you made upstream? I don't see a mention of that here.

Reply:
I did not submit it because I first wanted to have it tested a litle bit here. So far my changes work, no more issues.

Reply:
PR is submitted in the meantime https://github.com/amplitude/analytics-go/pull/56. Just wanted to make sure it works like expected before sumbitting it.

Reply:
Hint about the fork ist added in various places.